tools/cmake/Makefile

```diff
@@ -8,12 +8,13 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=cmake
 PKG_VERSION:=3.22.3
+PKG_VERSION_MAJOR:=$(word 1,$(subst ., ,$(PKG_VERSION))).$(word 2,$(subst ., ,$(PKG_VERSION)))
```

Comment:
`$(wordlist 1,2,$(subst ., ,$(PKG_VERSION)))` could be a more concise variant that does not require substituting twice.

Reply:
hmmmm, but the delimiter must be returned to a dot instead of a space...

Reply:
Ah right, of course. Nevermind then.